src/components/Article.jsx

```diff
@@ -1 +1,28 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({ title, date, text }) => {
```

Comment:
```suggestion
export const Article = ({ 
	title, 
	date, 
	text, 
}) => {
```
that one would be better